tests/format/js/import-assertions/bracket-spacing/jsfmt.spec.js

```diff
@@ -1,5 +1,11 @@
 run_spec(__dirname, ["babel"], {
-  bracketSpacing: false,
+  // [prettierx]: broken-out bracket spacing options
+  graphqlCurlySpacing: false,
+  exportCurlySpacing: false,
+  importCurlySpacing: false,
+  objectCurlySpacing: false,
+  typeCurlySpacing: false,
+  yamlBracketSpacing: false,
```

Comment:
ditto (Do we need *all* of these broken-out spacing options here?)

Reply:
ditto: (Whoops, forgot to whittle these down. Just pushed an update that keeps only necessary options.)